src/index.ts

```diff
@@ -49,15 +47,15 @@ const createFakeProduct = (): Product => {
     id: faker.datatype.number(),
     name: faker.commerce.productName(),
     description: faker.commerce.productDescription(),
-    price: faker.commerce.price(),
-    tags: [faker.commerce.productMaterial(), faker.commerce.color()],
+    price: parseInt(faker.commerce.price()),
+    tags: [faker.commerce.productMaterial(), faker.color.human()],
   };
 };
 
 const createRandomProduct = (email: string) => {
-  const userRole: string = users.find((user) => user.email === email).role;
-  const creatorRoles: string[] = ['creator'];
-  if (!creatorRoles.includes(userRole)) {
+  const userRole = users.find((user) => user.email === email)?.role;
+  const creatorRoles = ['creator'];
+  if (userRole == undefined || !creatorRoles.includes(userRole)) {
```

Comment:
Unnecessary clause, `.includes` method will return false even if userRole is undefined. However, if you'd like to keep the clause, I suggest you use `===` for comparing values, or you can use type coercion with the bang operator `if (!userRole || ...)`